vllm/core/scheduler.py

```diff
@@ -780,6 +782,8 @@ def _schedule_default(self) -> SchedulerOutputs:
         # Update swapped requests.
         self.swapped = remaining_swapped
         self.swapped.extend(running_scheduled.swapped_out)
+        preempted = (len(running_scheduled.preempted) +
+                     len(running_scheduled.swapped_out))
 
```

Comment:
Last question - is `swapped_out` the number that were swapped ON THIS ITERATION or the number that are currently swapped?

Reply:
it is the number of preempted requests in the previous iteration. our scheduler tracks recompute preemption and swap preemption in a separate values now